windows/lib/WixSDK.js

```diff
@@ -148,9 +149,18 @@ function(app_path, xwalk_path, meta_data, callback) {
     var file_ids = [];
 
     function MakeIdFromPath(path) {
-        // Only ASCII, digits, '.', '_' are allowed.
-        return path.replace(new RegExp('[^A-Za-z0-9_.]', 'g'), '_');
+        shasum = crypto.createHash('sha1');

```

Comment:
But does it make any difference? crypto.createHash() is still run on every invocation of MakeIdFromPath(), right? In that case I'd prefer not to have it global.